server.js

```diff
@@ -22,170 +22,25 @@
 
 const express = require('express')
 const app = express()
-const fetch = require('node-fetch')
 
 const config = require('./config')
-const get_tokens = require('./utils/get_tokens')
-const { createCodeVerifier, createCodeChallenge } = require('./utils/pkce')
-const { createState } = require("./utils/state")
+const authRouter = require('./routes/auth')
 
 // set the view engine to ejs
 app.set('view engine', 'ejs')
 
 app.use('/public/', express.static('./public'));
 
-// If you click on the URL in the log, "App running at http://localhost:8080", that will open the URL in your web browser.
-// In this case, we'll redirect you from the '/' route to the '/static' route to render the static HTML.
+app.use('/auth', authRouter)
+
 app.get('/', (req, res) => {
     res.redirect('/static')
 })
 
-// First plugin, renders static HTML
 app.get('/static', (req, res) => {
     res.render('pages/static')
 })
 
-// This example project doesn't include any storage mechanism (e.g. a database) for managing state.
-// Therefore, we use this as our 'storage' for the purposes of this example.
-// This method is NOT recommended for use in production systems.
```

Comment:
We definitely want this warning for folks, so they don't try to YOLO this code into production service.

Reply:
fixed in https://github.com/Banno/simple-plugin-example/pull/28/commits/c839045fe8a473445b2c08712c26e92fa4d60f1a